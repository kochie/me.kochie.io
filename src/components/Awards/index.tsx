import Image from 'next/image'
import Link from 'next/link'
import React, { forwardRef } from 'react'

const Img = (props, ref) => (
  <Image
    className="rounded-xl cursor-pointer"
    src="https://holopin.io/api/user/board?user=kochie"
    alt="@kochie's Holopin board"
    width={2428}
    height={764}
  />
)

const ImageRef = forwardRef(Img)

const Awards = () => (
  <div>
    <h2 className="text-gray-900 dark:text-white font-bold text-2xl mt-6 mb-4">
      Awards
    </h2>
    <div className="w-full">
      <Link href="https://holopin.io/@kochie">
        <ImageRef />
      </Link>
    </div>
  </div>
)

export default Awards
