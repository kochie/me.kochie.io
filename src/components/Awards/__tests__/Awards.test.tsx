import React from 'react'
import renderer from 'react-test-renderer'
import Awards from '..'

it('awards renders correctly', () => {
  const tree = renderer.create(<Awards />).toJSON()
  expect(tree).toMatchSnapshot()
})
