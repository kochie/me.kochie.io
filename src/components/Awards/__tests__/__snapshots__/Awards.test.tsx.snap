// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`awards renders correctly 1`] = `
<div>
  <h2
    className="text-gray-900 dark:text-white font-bold text-2xl mt-6 mb-4"
  >
    Awards
  </h2>
  <div
    className="w-full"
  >
    <span
      style={
        {
          "background": "none",
          "border": 0,
          "boxSizing": "border-box",
          "display": "inline-block",
          "height": "initial",
          "margin": 0,
          "maxWidth": "100%",
          "opacity": 1,
          "overflow": "hidden",
          "padding": 0,
          "position": "relative",
          "width": "initial",
        }
      }
    >
      <span
        style={
          {
            "background": "none",
            "border": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": "initial",
            "margin": 0,
            "maxWidth": "100%",
            "opacity": 1,
            "padding": 0,
            "width": "initial",
          }
        }
      >
        <img
          alt=""
          aria-hidden={true}
          src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%272428%27%20height=%27764%27/%3e"
          style={
            {
              "background": "none",
              "border": 0,
              "display": "block",
              "height": "initial",
              "margin": 0,
              "maxWidth": "100%",
              "opacity": 1,
              "padding": 0,
              "width": "initial",
            }
          }
        />
      </span>
      <img
        alt="@kochie's Holopin board"
        className="rounded-xl cursor-pointer"
        data-nimg="intrinsic"
        decoding="async"
        onError={[Function]}
        onLoad={[Function]}
        src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
        style={
          {
            "border": "none",
            "bottom": 0,
            "boxSizing": "border-box",
            "display": "block",
            "height": 0,
            "left": 0,
            "margin": "auto",
            "maxHeight": "100%",
            "maxWidth": "100%",
            "minHeight": "100%",
            "minWidth": "100%",
            "objectFit": undefined,
            "objectPosition": undefined,
            "padding": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": 0,
          }
        }
      />
      <noscript>
        <img
          alt="@kochie's Holopin board"
          className="rounded-xl cursor-pointer"
          data-nimg="intrinsic"
          decoding="async"
          loading="lazy"
          src="/_next/image?url=https%3A%2F%2Fholopin.io%2Fapi%2Fuser%2Fboard%3Fuser%3Dkochie&w=1200&q=75"
          srcSet="/_next/image?url=https%3A%2F%2Fholopin.io%2Fapi%2Fuser%2Fboard%3Fuser%3Dkochie&w=1200&q=75 1x"
          style={
            {
              "border": "none",
              "bottom": 0,
              "boxSizing": "border-box",
              "display": "block",
              "height": 0,
              "left": 0,
              "margin": "auto",
              "maxHeight": "100%",
              "maxWidth": "100%",
              "minHeight": "100%",
              "minWidth": "100%",
              "objectFit": undefined,
              "objectPosition": undefined,
              "padding": 0,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "width": 0,
            }
          }
        />
      </noscript>
    </span>
  </div>
</div>
`;
